### DISCUSSION TITLE
Speaker diarization pipeline in pyannote 2.0 ?

### DISCUSSION THREAD

**Discussion by Gabrielk99** [pyannote/pyannote-audio#739] [General]
Hi, thanks for the suport every ask that i do here.

In my recents issues you recommend me to use the segmentation model from pyannote 2.0, well, i read the readme and the hub page, and i have some questions.

My problem is the Diarization problem, i need the step of segmentation and the step of clustering and classification of speakers.

So

1) How i can use the model Segmentation from pyannote 2.0 with the Diarization Model?  In pyannote 2.0 has a tutorial with that? Sorry if this question has already been made, just let me know and i research for that.

**Comment by hbredin** [comment]
pyannote 2.0 does not have a working speaker diarization pipeline **yet**, only pyannote 1.1 does.

**Reply by Gabrielk99** [reply]
Oh but i can use pyannote 2.0 for the task? I see that Segmentation model return different waves to recognize the moment of activity voice, every one is a different speaker? 

I'm trying use the Segmentation model, and i cant print the output

this error mensage : ValueError: too many values to unpack (expected 2)

**Comment by hbredin** [comment]
The segmentation model can be used for diarization of 5s chunk but cannot be used directly to perform diarization for a whole file.
Please read [the related paper](https://www.isca-speech.org/archive/interspeech_2021/bredin21_interspeech.html) for a better understand of what this does.

**Reply by Gabrielk99** [reply]
Thanks, i will read the paper!!
